tests/sentry/incidents/test_subscription_processor.py

```diff
@@ -2187,9 +2194,9 @@ def send_crash_rate_alert_update(self, rule, value, subscription, time_delta=Non
                 else:
                     denominator = count
                     numerator = int(value * denominator)
-            session_status = resolve_tag_key(org_id, "session.status")
-            tag_value_init = resolve_weak(org_id, "init")
-            tag_value_crashed = resolve_weak(org_id, "crashed")
+            session_status = resolve_tag_key(UseCaseKey.RELEASE_HEALTH, org_id, "session.status")
+            tag_value_init = resolve_weak(UseCaseKey.PERFORMANCE, org_id, "init")
```

Comment:
This is session status, please set that use case to release health and the same with following line

Reply:
fixed in a2e7352dd2